apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx

```diff
@@ -64,3 +64,59 @@ Like any other project using Cypress, you'll follow standard practices, includin
 ![checks-integration-tests](https://vtexhelp.vtexassets.com/assets/docs/src/custom-test-checks___583ac9b78e43ebdafde93a735a47d61b.png)
 
 6. Review the results to check the execution status of your custom integration tests.
+
+---
+
+## Exclusive execution of your custom Cypress tests
+
+With continuous development, some storefront projects may have changes in the components and/or pages's code that cause errors in the out of the box end-to-end tests that are run for all FastStore projects.
+
+For these cases, there is the option to override the FastStore's out of the box tests using the same name as the original test file:
+
+```js filename="plp.test.js"
+import { cypress } from '../../faststore.config'
+
+const { pages } = cypress
+
+describe('Custom PLP tests', () => {
+  it('visit collection page', () => {
+    cy.visit(pages.collection)
+    cy.get('[data-testid="product-gallery"]').should('exist').and('be.visible')
+  })
+})
+```
+
+Some projects may prefer exclusive execution of its custom Cypress tests. Although it is not our recommendation, for projects using Cypress version 12.x or higher there is the possibility of running the **Integration Tests** check only for your custom tests.
```

Comment:
```suggestion
Although we recommend keeping the default Cypress tests, it is possible to run the Integration Tests check exclusively for your custom tests for projects using Cypress version 12.x or higher."
```

or 

Although we do not recommend exclusive execution of custom Cypress tests for some projects, it is possible to run the Integration Tests check exclusively for your custom tests for projects using Cypress version 12.x or higher.

Reply:
@hellofanny and @lucasfp13, I combined our suggestions and added a callout suggestion above :) let me know what you think.